Подскажите по шаблонам: что я делаю не так?

http://take.ms/yAQRP
- сделал файл шаблона для css
- указал его в .bemrc
- вызвал bem create
- результат из шаблона по умолчанию

Что я сделал не так?
(винда, зависимости проекта ставил вчера)

Желание: использовать свои шаблоны. Ибо у меня отступ в 2 пробела и LF-строки (хотя я и в винде). Плюс, может оказаться полезным добавлять что-то в шаблон, к примеру, в препроцессорные файлы добавлять импорты файлов с общими переменными проекта или комменты.

https://github.com/nicothin/test_gulp-bem/tree/add-project-stub--templates — ветка с попыткой (и фейлом)

Даже если это в созданном мной сейчас виде заработает, оно будет малоюзабельно, ибо https://github.com/nicothin/test_gulp-bem/commit/5bc1e095dbbe6702ad2235573ffe433e7130055e#diff-eab13dbc16d84c282da4d6469012915eR22 — путь указан с обратными слешами (винда же).
Если использовать обычные слеши, выпадает с ошибкой:
```
nicothin@nicwin10 MINGW64 /d/projects/test_gulp-bem (add-project-stub--templates)
$ node_modules/.bin/bem create -l common.blocks/ -b test-1 -T css
(node:4052) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Cannot find module '.bem\templates\css.js'
```

--- a/lib/template.js
+++ b/lib/template.js
@@ -17,7 +17,7 @@ module.exports = function getTemplate(tech, options) {
 
     if (!templatePath) {
         for (var i = 0; i < possiblePaths.length; i++) {
-            var possibleTemplatePath = path.join(possiblePaths[i], techId + '.js');
+            var possibleTemplatePath = path.resolve(possiblePaths[i], techId + '.js');
 
             if (fs.existsSync(possibleTemplatePath)) {
                 templatePath = possibleTemplatePath;

